quest_test/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+markers =
+    integration: mark a test as an integration test
+    unit: mark a test as a unit test
```

Comment:
These allow for certain tests to be run from the command line with `pytest -m <marker>`. You can create a configuration in PyCharm that mimics this command

Reply:
You can also exclude certain markers by running `pytest -m "not <marker>" <file>`. This will allow users to ignore integration tests if they want to.